live-examples/js-examples/regexp/meta.json

```diff
@@ -66,6 +66,13 @@
             "title": "JavaScript Demo: RegExp.prototype.global",
             "type": "js"
         },
+        "regexPrototypeDotAll":{
```

Comment:
This should be `"regexpPrototypeDotAll"` and this file should be ordered, so it should go after `"regexpPrototype@@Split"`